my-app/src/Container/constantsContainer/constantContainer.js

```diff
@@ -0,0 +1,32 @@
+import avatar1 from "../../assets/img/chat_avatar_01.jpg";
+import avatar2 from "../../assets/img/chat_avatar_02.jpg";
+import avatar3 from "../../assets/img/chat_avatar_03.jpg";
+import avatar4 from "../../assets/img/chat_avatar_04.jpg";
+import avatar5 from "../../assets/img/chat_avatar_05.jpg";
+
+export const persons = [
```

Comment:
all constants should be in SUCH_STYLE

Reply:
fixed the error as follows:
avatar4  -->  avatar_4 
firstMessages --> first_messages 
etc.